src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -153,9 +153,7 @@ public function supportsNormalization(mixed $data, string $format = null /* , ar
      */
     public function normalize(mixed $object, string $format = null, array $context = [])
     {
-        if (!isset($context['cache_key'])) {
```

Comment:
I'd keep this if to avoid calling a method each time. Indeed, calling a method could be costly and we are in the hot path here.